### DISCUSSION TITLE
Exportsymbol Card Style Compact und Lite

### DISCUSSION THREAD

**Discussion by ollelinse** [slipx06/sunsynk-power-flow-card#240] [Q&A] [answered]
![GridSell1](https://github.com/slipx06/sunsynk-power-flow-card/assets/62350400/c503b593-b447-4790-96a3-85639216cf6c)

Bei Card Style Full passt das Exportsymbol gut. Bei Lite und Compact steht der Pfeil in Richtung Inverter. Lässt sich das austauschen damit es dann etwa so aussieht?

![GridSell2](https://github.com/slipx06/sunsynk-power-flow-card/assets/62350400/affdc5e1-4c22-4d9b-8c5c-d38541fe9653)

Das 2. Fragezeichen steht für diesen kleingedruckten Wert (-132 W). Lässt sich dieser Wert komplett ausblenden? Ich habe grid_ct_power_172 belegt, die optionalen Entities freigelassen.

Danke

**Comment by slipx06** [comment, marked as answer]
Thanks for the suggestion. I'll look at changing the icon. 
You should set `three_phase: false` to hide that value

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/d3210298-bf2b-41fe-82a1-b74bf9e61f9e)

**Reply by ollelinse** [reply]
Vielen Dank, das hat gut geklappt.

**Comment by slipx06** [comment]
Added to 4.14.0 https://github.com/slipx06/sunsynk-power-flow-card/releases/tag/v4.14.0

**Reply by ollelinse** [reply]
Großartige Arbeit, vielen Dank dafür. Aber sehen kann ich die Veränderungen nach dem Update auf 4.14 noch nicht.

**Reply by slipx06** [reply]
You can try clear your browser cache and force a hard reload. F12 and then right click the refresh button select Hard Reload

**Reply by ollelinse** [reply]
Alles klar, beim 3, Versuch hat es geklappt, Die neuen Funktionen sind hervorragend umgesetzt. Ganz großes Lob!
